drivers/bluetooth/hci/userchan.c

```diff
@@ -289,9 +332,24 @@ static void add_btuserchan_arg(void)
 
```

Comment:
Please also update the help message in line 283 / 326 to include the new possible input